source/engine/physics/PhysicsBody.cpp

```diff
@@ -122,7 +108,7 @@ void PhysicsBody::ApplyImpulse(const Vector3& impulse)
 void PhysicsBody::ApplyImpulse(DirectX::FXMVECTOR impulse)
 {
     NC_PHYSICS_ASSERT(!m_properties.isKinematic, "Attempting to move immovable object");
```

Comment:
This is a nit, and I didn't notice it the first time around - but will you change these error messages to "Cannot move an immovable object" - I don't like "Attempting" here because it could be construed as: "we failed somewhere along the attempt to move an immovable object"

Reply:
Yes, fixed